src/main/java/com/khomishchak/ws/services/integration/whitebit/configs/WhiteBitWebConfig.java

```diff
@@ -34,10 +35,16 @@ HttpClient whiteBitApiHttpClient(
 
     @Qualifier("WhiteBitApiWebClient")
     @Bean
```

Comment:
Wrong usage of `@Qualifier`. This annotation is only for injection purposes. So if you want to set a bean id manually you should provide it as a parameter of `@Bean` instead.
```suggestion
    @Bean("WhiteBitApiWebClient")
```